vendor/github.com/beorn7/perks/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (C) 2013 Blake Mizerany
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
```

Comment:
I'm glad that github hides the vendored files from the diff by default.  I also understand that we don't really have any control over the content of these files (that's the point of vendoring - to copy in a frozen point in time of the dependencies we used).  Do we need to do any automated checking of whether vendored tools have correct Licensing compatible with the rest of our code base?  (Technically, we should already require that everything we depend on was the right license - but now that it is explicitly in our repo, it becomes more obvious if we failed at that by pulling in something with an incompatible license)

Reply:
We don't usually do this in our repos but I'm not sure if legal has anything to say about it? Something like snyk might check that automatically but I'm not sure off the top of my head